content/docs/command-reference/status.md

```diff
@@ -156,24 +156,24 @@ that.
 
 ```dvc
 $ dvc status
-bar.dvc:
-        changed deps:
-                modified:      bar
-        changed outs:
-                not in cache:      foo
+bar:
+	changed deps:
+		modified:           bar
+	changed outs:
+		not in cache:       foo
 foo.dvc:
-        changed outs:
-                deleted:      foo
-        changed checksum
+	changed outs:
+		deleted:            foo
+	changed checksum
 prepare.dvc:
-        changed outs:
-                new:      bar
-        always changed
+	changed outs:
+		new:                bar
+	always changed
```

Comment:
foo/prepare.dvc ?

Reply:
Maybe foo? Not sure, need to run the whole exp.

Reply:
Updated this example. The first two are stages and and last one is a `.dvc` file. Since its a general example(first one) so both stags and `.dvc` file are included.